### ISSUE TITLE
SUGGESTION: Synccit custom URL

### ISSUE THREAD

**Issue by RubenNL** [ccrama/Slide#3122]
Slide version:6.2.2
Android version:8.1.0

I run my own synccit server, on a custom URL. I have no experience with java, but i have found the url in the code, its at https://github.com/ccrama/Slide/blob/0541b746f09cc4ee61f2957e466840a568209015/app/src/main/java/me/ccrama/redditslide/Synccit/SynccitTask.java#L16.

Is it possible to make this a setting?

**Comment by Alexendoo** [comment]
Yes, that is certainly possible. There'd need to be a new method in SynccitTask.java called `getEndpoint` or something along those lines. They'd work the same way as the current `getUsername` etc

**Comment by RubenNL** [comment]
I wasn't able to create a getUrl function or something, because it had to be called on creation of the HttpPostTask class.

**Comment by Kritip123** [comment]
### @Alexendoo 
I want to work on this issue. This is going to be my first proper open-source contribution. I need help from the members of this project. I am really looking forward to contributing to this. Please tell me more about the issue and tell me if there are some other groups or servers where I can join and get familiar with the codebase by discussing with the members.

**Comment by GiwrgosTsifoutis** [comment]
Hello i would like to tackle this issue. This is my fist time contributing to OSS and this will be an assignment for my university. If this issue has been resolved could you lend another issue that needs to be fixed? Thanks a lot!
